packages/react/src/components/LinkWithIcon/LinkWithIcon.js

```diff
@@ -17,10 +18,13 @@ const { prefix } = settings;
 /**
  * LinkWithIcon component.
  */
-const LinkWithIcon = ({ children, href, ...props }) => {
+const LinkWithIcon = ({ children, href, inverse, ...props }) => {
   return (
     <div
-      className={`${prefix}--link-with-icon__container`}
+      className={classnames(
+        `${prefix}--link-with-icon__container`,
+        inverse ? `${prefix}--link-with-icon__container__inverse` : null
```

Comment:
```suggestion
        { [`${prefix}--link-with-icon__container__inverse`]: inverse }
```

Just another way to achieve the same thing (I think the original code is good, too).